pgo/cmd/create.go

```diff
@@ -28,7 +28,7 @@ var AllNamespaces bool
 var ContainerResources string
 var ReplicaStorageConfig, StorageConfig string
 var CustomConfig string
-var ArchiveFlag, DisableAutofailFlag, PgbouncerFlag, MetricsFlag, BadgerFlag bool
+var ArchiveFlag, DisableAutofailFlag, EnableAutofailFlag, PgbouncerFlag, MetricsFlag, BadgerFlag bool
```

Comment:
If you need to make this change here, then don't you also need to address it in `createCluster(...)` in `pgo/cmd/cluster.go`?

This would also bring me back to my original concern about two flags for the same thing on `create`. :thinking:

Reply:
Autofail is enabled by default on create. We only need to disable it there.

Reply:
I just put it there for consistency where all the other flags were defined.